builtin/app/go/data/common/dev/Vagrantfile.tpl

```diff
@@ -74,13 +74,18 @@ fstype=$(find /opt/gopath -mindepth 0 -maxdepth 0 -type d -printf "%F")
 find /opt/gopath -fstype ${fstype} -print0 | xargs -0 -n 100 chown vagrant:vagrant
 
 ol "Setting up PATH..."
-echo 'export PATH=/opt/gopath/bin:/usr/local/go/bin:$PATH' >> /home/vagrant/.bashrc
-echo 'export GOPATH=/opt/gopath' >> /home/vagrant/.bashrc
+echo 'export PATH=/opt/gopath/bin:/usr/local/go/bin:$PATH' >> /home/vagrant/.profile
+echo 'export GOPATH=/opt/gopath' >> /home/vagrant/.profile
```

Comment:
This change was necessary so that below when we call bash this is loaded. The Ubuntu `.bashrc` will bail if its not interactive.